### PR TITLE
capitalize UDP and TCP correctly

### PR DESCRIPTION
**Pull request by jancborchardt**
I guess that is the correct capitalization, see https://github.com/nextcloud/spreed/pull/188/files#r92625064

Please review @nextcloud/spreed

### CODE REVIEW

**Comment by jancborchardt** [comment]
Just noticed the HTML errors there (see red marked `>`) … was that intended @MorrisJobke?

**Comment by nickvergessen** [comment]
I guess I failed with copy+paste there, just remove them

**Comment by nickvergessen** [comment]
I accidently added them in https://github.com/nextcloud/spreed/pull/188/files#r92625064

File: templates/settings-admin.php [hunk 1]
```diff
@@ -33,13 +33,13 @@
...
-                    <?php p($l->t('udp and tcp')) ?><</option>
+                    <?php p($l->t('UDP and TCP')) ?><</option>
                 <option value="udp"
                     <?php p($_['turnServerProtocols'] === 'udp' ? 'selected' : '') ?>>
-                    <?php p($l->t('udp only')) ?><</option>
+                    <?php p($l->t('UDP only')) ?><</option>
                 <option value="tcp"
                     <?php p($_['turnServerProtocols'] === 'tcp' ? 'selected' : '') ?>>
-                    <?php p($l->t('tcp only')) ?><</option>
+                    <?php p($l->t('TCP only')) ?><</option>
```
Comment (MorrisJobke) [inline templates/settings-admin.php]: While on it also fix the wrong `<` ;)

**Comment by jancborchardt** [comment]
Fixed the HTML issues and also the indentation so that error is less likely to happen.

**Pull request merged**